packages/docs/docs/docs/migrate_from_1.x_to_2.x.md

```diff
@@ -0,0 +1,117 @@
+---
+id: migrate_from_1.x_to_2.x
+title: Migrating to 2.x
+---
+
+In this section, you'll find everything you need to know if you're coming from ZeppelinOS 1.x and want to start using 2.x.
+
+## Terminology and fundamentals
+
+2.x's most fundamental change is that it expands on zOS's linking features. In 1.x, you were able to link your projects to what used to be called an "on-chain standard library". This "stdlib" was basically a pre-deployed version of OpenZeppelin, which gave your projects instant access to hundreds of standard implementations without having to compile or deploy any code whatsoever. This proved to be a very powerful, innovative concept, but we realized that it could be taken much further... In 1.x, you could connect to different versions of this "standard library" but there was just this ONE "standard library" that your application could link to. 2.x takes this to a whole new level; you can now connect your project to multiple "standard libraries", but that's not all, this makes the entire concept of what used to be a "standard library" evolve to the concept of an "EVM package".
```

Comment:
> This "stdlib" was basically a pre-deployed version of OpenZeppelin

Not sure we want to frame it like this. Even though the only stdlib available _was_ OZ, the intention was to have different competing stdlibs. I'd rephrase it as "a pre-deployed version of a set of common reusable contracts, like OZ", or similar.